sql/plan/alter_default.go

```diff
@@ -114,29 +114,30 @@ func (d *AlterDefaultSet) Expressions() []sql.Expression {
 	return append(transform.WrappedColumnDefaults(d.targetSchema), expression.WrapExpressions(d.Default)...)
 }
 
-func (d AlterDefaultSet) WithExpressions(exprs ...sql.Expression) (sql.Node, error) {
```

Comment:
Why were these changed to use pointer receivers?

Reply:
All the other methods are pointer reciever, and Goland was complaining.